Code/GraphMol/ScaffoldNetwork/catch_tests.cpp

```diff
@@ -763,6 +763,55 @@ TEST_CASE("Github #3177: seg fault with null molecules", "[bug]") {
   }
 }
 
+TEST_CASE("GitHub #3153: Kekulization error in molecules with aromatic C+", "[bug]") {
```

Comment:
The tests that you've added are good in the sense that they make sure that you can add these molecules to the scaffold network, but they aren't looking at the correctness of the result.

Please add, to each of the SECTIONS, a check that the expected results are there.
For example, in the aromatic carbocation example, you should check that the canonical SMILES corresponding to `[CH+]1C=CC(CC2=CC=CC2)=CC=C1` is in the `net.nodes` vector.